### PR TITLE
releasenotes/notes: update releasenotes

### PR DESCRIPTION
**Pull request by gbbr**

### CODE REVIEW

**Review by olivielpeau — COMMENTED** [review]
Thanks, these changes look good to me, but could you update the CHANGELOG file accordingly?

(we don't have a good way to update the CHANGELOG file when release notes are updated unfortunately, so once the release note entries have been added to the actual CHANGELOG, any update to the release notes have to be manually reflected in the CHANGELOG)

File: releasenotes/notes/apm-rm-service-writer-8818e02f95020d2c.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #
 # Each section note must be formatted as reStructuredText.
 ---
-upgrade:
+enhancements:
```
Comment (olivielpeau) [inline releasenotes/notes/apm-rm-service-writer-8818e02f95020d2c.yaml]: this is still listed as an `Upgrade` item in the changelog :)

Comment (gbbr) [reply releasenotes/notes/apm-rm-service-writer-8818e02f95020d2c.yaml]: Whoops, fixed.

**Review by gbbr — COMMENTED** [review]
I think we should be good now. Thank you for reviewing thoroughly. 🙇 PTAL

**Review by olivielpeau — APPROVED** [review]

**Comment by olivielpeau** [comment]
merging, will update the github releases accordingly, and backport on `6.13.x`

**Pull request merged**